arrow-data/src/test/kotlin/arrow/data/EitherTest.kt

```diff
@@ -32,7 +32,7 @@ class EitherTest : UnitSpec() {
 
         testLaws(
             EqLaws.laws(eq<Either<String, Int>>(), { Right(it) }),
-            ShowLaws.laws { Right(it) },
+            ShowLaws.laws(show<Either<String, Int>>(), eq<Either<String, Int>>(), { Right(it) }),
```

Comment:
Either.show() and Either.eq() have to be shorter :D

Reply:
What do you mean?